modules/ci/.circleci/circleci.tmpl

```diff
@@ -0,0 +1,34 @@
+---
+delimiters
+  - "{{"
+  - "}}"
+---
```

Comment:
Optional frontmatter support. We don't need it right away but it does add a lot of flexibility.

Reply:
Why would these be specified here if they are already in the module yaml? To be able to override on a per-file basis?

Reply:
yup all front-matter variables will be for per file overrides. We totally don't need this feature right away but I just wanted to include it as part of the draft.